CONTRIBUTING.md

```diff
@@ -142,6 +142,15 @@ cd docs/_build/html/
 open index.html
 ```
 
+## Write integration tests
+1. If adding functionality to a new part of the library, create new file with a class that inherits `IntegrationTestCase` from `tests.integration.integration_test_case` to store all individual tests under (ex: `class TestWallet(IntegrationTestCase)`). Otherwise, add to an existing file.
+2. Create an async (likely) or sync (if specifically using an individual sync client) function that contains logic for the test
```

Comment:
```suggestion
2. Create an async function for each test case (unless the test is only being used for the sync client)
```
Parentheses pull the reader out of the flow of the sentence for a (hopefully quick) aside. It can be distracting to have them up front. Also, it's a bit easier to follow directions if the default path is clearly outlined first.